### PR TITLE
Xeno abilities cleanup and change from singular to plural.

### PR DESCRIPTION
**Pull request by Rohesie**
Code cleaning, removed a duplicated file and changing the xeno feedback from singular to plural.

A start to the transition of singular into plural entities for hivemind members. The goal is to improve a little the RP ambience from the xenos.

:cl:
tweak: The xenomorph hivemind members now use plural forms to refer to themselves. They have a hard time grasping concepts such as individuality. All are one, one is many.
/:cl:

### CODE REVIEW

File: code/datums/actions/xeno_action.dm [hunk 1]
```diff
@@ -208,7 +208,7 @@
 
 /datum/action/xeno_action/activable/can_use_action(silent = FALSE, override_flags, selecting = FALSE)
 	if(selecting)
-		return ..(silent, XACT_IGNORE_COOLDOWN|XACT_IGNORE_PLASMA|XACT_USE_STAGGERED)
```
Comment (spookydonut) [inline code/datums/actions/xeno_action.dm]: This is for selecting abilities that are on cooldown, anything thats activable shouldn't do anything when selected.

File: code/datums/actions/xeno_action.dm [hunk 2]
```diff
@@ -46,57 +46,57 @@
 
 	if(!CHECK_BITFIELD(flags_to_check, XACT_IGNORE_COOLDOWN) && !action_cooldown_check())
 		if(!silent)
-			to_chat(owner, "<span class='warning'>You can't use [ability_name] yet, wait [cooldown_remaining()] seconds!</span>")
+			to_chat(owner, "<span class='warning'>We can't use [ability_name] yet, we must wait [cooldown_remaining()] seconds!</span>")
```
Comment (spookydonut) [inline code/datums/actions/xeno_action.dm]: revert this, outside the scope.

**Comment by spookydonut** [comment]
split off the grammar changes into a separate pr.

**Comment by Rohesie** [comment]
Did the contrary, split off the fixes to another PR.

File: code/datums/actions/xeno_action.dm (hunk 2 above)
Comment (Rohesie) [reply code/datums/actions/xeno_action.dm]: I use my reverse card.

**Comment by Rohesie** [comment]
The invalid tag may go now.

**Comment by spookydonut** [comment]
I feel this makes messages less clear and is inconsistent with other feedback messages being directed at the player not the IC character.

**Comment by Rohesie** [comment]
We can get it ran through #lore .
I'd like to give some flavor to our xenos and I think this kind of guideline and quirk makes them more interesting.

**Comment by spookydonut** [comment]
> 
> 
> We can get it ran through #lore .
> I'd like to give some flavor to our xenos and I think this kind of guideline and quirk makes them more interesting.

This isn't about lore or IC flavour these are OOC feedback messages to the user.

**Comment by Rohesie** [comment]
These OOC messages already use IC terminology.

**Comment by Metroidlover** [comment]
As a side note, EVERYTHING in the game screen should be put under flavor consideration, you'll see everything whether you like it or not.

**Comment by Rohesie** [comment]
This is ready for merge.
While controversial, it has received way more support than opposition.

**Pull request merged**